lib/apiAiResponseFormatter.js

```diff
@@ -13,24 +13,36 @@ const contextOut = (shownShelterIds) => (
     ] : []
 );
 
-const responseObject = (text, shownShelterIds = []) => (
+const responseObject = (text, shownShelterIds = [], mapsUrl) => (
   {
     speech: text,
     displayText: text,
+    mapsUrl: mapsUrl,
```

Comment:
The object here is a format given by API.AI. You can see what it supports in the "Response" tab of this page: https://api.ai/docs/fulfillment

I don't think adding mapsUrl here will break it... but it certainly will not do anything useful here.

Reply:
Cool, will fix that.